spec/rails_helper.rb

```diff
@@ -54,4 +55,12 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  config.before(:each, type: :system) do
+    driven_by :rack_test
+  end
+
+  config.before(:each, type: :system, js: true) do
+    driven_by :selenium_chrome_headless
+  end
```

Comment:
These blocks define which driver to use based on whether or not your spec uses JavaScript.